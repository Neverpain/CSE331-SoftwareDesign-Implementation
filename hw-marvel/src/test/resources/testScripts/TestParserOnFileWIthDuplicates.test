# Tests that the makeGraph method on a given file with duplicate lines does not
# contain two of the same nodes or reflexive edges, prints them out

LoadGraph g duplicateGraph.csv

ListNodes g
ListChildren g n1
ListChildren g n2