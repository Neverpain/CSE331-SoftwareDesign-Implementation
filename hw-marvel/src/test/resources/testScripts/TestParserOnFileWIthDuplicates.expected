# Tests that the makeGraph method on a given file with duplicate lines does not
# contain two of the same nodes or reflexive edges, prints them out

loaded graph g

g contains: n1 n2
the children of n1 in g are: n2(same)
the children of n2 in g are: n1(same)